projects/media-viewer/src/lib/media-viewer/annotations/annotations.module.ts

```diff
@@ -0,0 +1,25 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { FormsModule } from '@angular/forms';
+import { HttpClientModule } from '@angular/common/http';
+import { NgtUniversalModule } from '@ng-toolkit/universal';
+import { AnnotationApiService } from './annotation-api.service';
+
+@NgModule({
+  imports: [
+    NgtUniversalModule,
+    CommonModule,
+    FormsModule,
+    HttpClientModule
+  ],
+  declarations: [
+  ],
+  entryComponents: [
+  ],
+  providers: [
+    AnnotationApiService
+  ],
+  exports: [
+  ]
+})
+export class AnnotationsModule { }
```

Comment:
i did this however, as my changes arent here anymore, maybe you want to add this module in the media-viewer module?

Reply:
ok